docs/lang/articles/misc/install.md

```diff
@@ -35,11 +35,11 @@ sidebar_position: 0
   ERROR: No matching distribution found for taichi
   ```
 
-  - Make sure you're using Python version 3.6/3.7/3.8/3.9:
+  - Make sure you're using Python version 3.6/3.7/3.8/3.9/3.10:
 
     ```bash
     python3 -c "print(__import__('sys').version[:3])"
-    # 3.6, 3.7, 3.8 or 3.9
+    # 3.6, 3.7, 3.8, 3.9, or 3.10
```

Comment:
```suggestion
    # 3.6, 3.7, 3.8, 3.9 or 3.10
```

Reply:
@rexwangcc Thanks, I actually vote for keeping the [Oxford Comma ](https://prowritingaid.com/Oxford-Comma). ;/

Reply:
Cool, I cannot argue with that :P